wyvern/experimentation/client.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from .providers.eppo_provider import EppoExperimentationClient
+
+
+class ExperimentationClient:
+    def __init__(self, provider_name: str):
+        """
+        Initializes the ExperimentationClient with a specified provider.
+
+        Args:
+        - provider_name (str): The name of the experimentation provider (e.g., "eppo").
+        """
+        if provider_name == "eppo":
+            self.provider = EppoExperimentationClient()
+        else:
+            raise ValueError(f"Unsupported provider: {provider_name}")
```

Comment:
Let's make all the error we raise within pipeline code should be a [WyvernError](https://github.com/Wyvern-AI/wyvern/blob/main/wyvern/exceptions.py#L5)